libs/auth/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./auth";
+export * from "./callback";
+export { default } from "./callback";
```

Comment:
Why this pattern?

Reply:
It's a bit weird, but it's meant to be exposed as a page via next (meaning it has to be a default export)

Reply:
Ah, and `export *` doesn't include `default`. Weird little gotcha.